drop-in/src/main/java/com/adyen/checkout/dropin/ui/paymentmethods/PaymentMethodModel.kt

```diff
@@ -9,6 +9,10 @@
 package com.adyen.checkout.dropin.ui.paymentmethods
 
 data class PaymentMethodModel(
-    val type: String,
-    val name: String
-)
+    val type: String?,
```

Comment:
Why did you decide to make this nullable when it wasn't before? Maybe the logic is not in the best place but we check for nullability in `PaymentMethodsListViewModel:85`

Reply:
I... don't know 😆  will check